tests/baselines/reference/completionsStringMethods.baseline

```diff
@@ -6,6 +6,7 @@
       "name": "1"
     },
     "completionList": {
+      "flags": 0,
```

Comment:
Skimming, I couldn't find one that wasn't 0?

Reply:
It doesn't seem like any of the baselines have a value other than 0, no, but there is a unit test that has one of the flags set.